app/models/work_statistic.rb

```diff
@@ -125,16 +125,16 @@ def recalculate_from_hash(stats=nil)
 
     self[:total_pages] = stats[:total]
 
-    self[:transcribed_pages]  = stats[:transcription][Page::STATUS_TRANSCRIBED] || 0
-    self[:corrected_pages]    = stats[:transcription][Page::STATUS_TRANSCRIBED] || 0
-    self[:blank_pages]        = stats[:transcription][Page::STATUS_BLANK] || 0
-    self[:annotated_pages]    = stats[:transcription][Page::STATUS_INDEXED] || 0
-    self[:needs_review]       = stats[:transcription][Page::STATUS_NEEDS_REVIEW] || 0
-
-    self[:translated_pages]     = stats[:translation][Page::STATUS_TRANSLATED] || 0
-    self[:translated_blank]     = stats[:translation][Page::STATUS_BLANK] || 0
-    self[:translated_annotated] = stats[:translation][Page::STATUS_INDEXED] || 0
-    self[:translated_review]    = stats[:translation][Page::STATUS_NEEDS_REVIEW] || 0
+    self[:transcribed_pages]  = stats[:transcription]['transcribed'] || 0
```

Comment:
I'll modify this. But essentially, the reason I did this is the function used to group `stats` is grouping it by key_name in string literals. So that instead of `review` it uses `needs_review` string literal for the hash map. Let me think more on this